q2_annotate/busco/utils.py

```diff
@@ -268,3 +268,13 @@ def _get_mag_lengths(bins: Union[MultiMAGSequencesDirFmt, MAGSequencesDirFmt]):
             seq = skbio.io.read(mag_fp, format="fasta")
             lengths[mag_id] = sum([len(s) for s in seq])
         return pd.Series(lengths, name="length")
+
+
+def _validate_parameters(lineage_dataset, auto_lineage,
+                         auto_lineage_euk, auto_lineage_prok):
+    if all(value is None for value in locals().values()):
```

Comment:
Could you also have a check to test if both `lineage_dataset` and one of the `auto` options were provided? When that happens BUSCO throws an error as they are incompatible.